docs/reference/server-config.md

```diff
@@ -359,6 +361,29 @@ Example:
 	</tr>
 </table>
 
+## Hot logging level reload
+We don't support to completely reload the whole configuration yet at present. What we support for now is:
+- increase logging level by signaling `SIGUSR1`
+- decrease logging level by signaling `SIGUSR2`
+
+Example:
+
+To increase logging level
+```
+$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
```

Comment:
Actually NM, you are correct, it looks like the `docker kill` process seems to only send it to PID1, which is the `sh -c ./migrations/migrate.sh && notary-server -config=fixtures/server-config.json`  :|  which doesn't seem to do anything.

Exec-ing in also seems to have issues for me?
```
$ docker exec notary_server_1 sh
/go/src/github.com/docker/notary # kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
sh: invalid number 'root'
sh: invalid number 'root'
/go/src/github.com/docker/notary # ps aux 
PID   USER     TIME   COMMAND
    1 root       0:00 sh -c ./migrations/migrate.sh && notary-server -config=fixtures/server-config.json
  115 root       0:00 notary-server -config=fixtures/server-config.json
  124 root       0:00 sh
  129 root       0:00 sh
  150 root       0:00 ps aux
```

Looks like if you exec in, the first column is the PID.